test/e2e/e2e_test.go

```diff
@@ -104,6 +104,9 @@ var _ = BeforeSuite(func() {
 		HaveLen(1),
 		ContainElement(Not(BeZero())),
 	))
+	_, err := fetchCatalogSourceOnStatus(ctx.Ctx().OperatorClient(), "operatorhubio-catalog", operatorNamespace, catalogSourceRegistryPodSynced)
```

Comment:
Thank you for your contribution -- I think this makes sense and is a safe addition. 

/lgtm